react/package.json

```diff
@@ -16,6 +16,7 @@
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.3",
     "@types/react-table": "^7.0.23",
+    "copy-to-clipboard": "^3.3.1",
```

Comment:
What's the reasoning behind introducing another dependency instead of directly using the native [synchronous](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) or [asynchronous](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText) APIs?

Reply:
With this package, I only need to write one line of code: `copy(toCopy)`.
Without it, I will need to create a DOM object in order to select and copy something, so there would be more code: 
`const dummy = document.createElement("input");`
           ` document.body.appendChild(dummy);`
       `     dummy.setAttribute('value', toCopy);`
          `  dummy.select();`
         `   document.execCommand("copy");`
          `  document.body.removeChild(dummy);`
They both works, so I can switch to use the native API if you think it's better.

Reply:
That is pretty awful but because we don't care about IE at all, I think you can use the [newer async API](https://caniuse.com/mdn-api_clipboard_writetext) that's well-supported and also one line:
```js
await navigator.clipboard.writeText("foo");
```